ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+I'm submitting a ... (check one with "x")
+- [ ] bug report
+- [ ] feature request
```

Comment:
Please comment the comment with `<!-- (check one with "x") -->`

See: https://raw.githubusercontent.com/facebook/react/master/.github/ISSUE_TEMPLATE.md

Reply:
Probably a few other options we need to add here, one of which is simply "Question" (you'll have to look over the issues to figure out any others)

Reply:
I read through a few pages of recent issues and checked the labels you've been using..  I think bug/feature/question covers it pretty well; let me know if you want to add any others.